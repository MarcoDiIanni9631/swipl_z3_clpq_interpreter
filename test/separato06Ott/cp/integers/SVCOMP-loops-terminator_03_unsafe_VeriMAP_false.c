# 1 "MAP/UNSAFE-exbench/SVCOMP13-loops-terminator_03_unsafe.tmp.c"
# 1 "<command-line>"
# 1 "MAP/UNSAFE-exbench/SVCOMP13-loops-terminator_03_unsafe.tmp.c"
# 18 "MAP/UNSAFE-exbench/SVCOMP13-loops-terminator_03_unsafe.tmp.c"
/*
void __VERIFIER_assert(int cond) {
  if (!(cond)) {
    ERROR: goto ERROR;
  }
  return;
}
//extern int __VERIFIER_nondet_int();
*/

main()
{
  int x=__VERIFIER_nondet_int();
  int y=__VERIFIER_nondet_int();

  if (y>0)
  {
    while(x<100)
    {
      x=x+y;
     }
  }
  __VERIFIER_assert(y<=0 || (y<0 && x>=100));
}
