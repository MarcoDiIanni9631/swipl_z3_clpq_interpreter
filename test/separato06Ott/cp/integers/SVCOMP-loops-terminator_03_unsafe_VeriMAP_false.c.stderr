MAP/UNSAFE-exbench/SVCOMP13-loops-terminator_03_unsafe.tmp.c:40: Warning: Body of function main falls-through. Adding a return statement
